assistant/src/main/java/com/ibm/watson/assistant/v2/model/BaseEnvironmentOrchestration.java

```diff
@@ -21,58 +22,8 @@ public class BaseEnvironmentOrchestration extends GenericModel {
   @SerializedName("search_skill_fallback")
   protected Boolean searchSkillFallback;
 
-  /** Builder. */
-  public static class Builder {
```

Comment:
is the builder no longer required here?

Reply:
Good eye 👁️. These models are response models so they do not need builders. Not sure why they had builders before but they are not necessary